.zenodo.json

```diff
@@ -0,0 +1,109 @@
+{
+  "creators": [
+    {
+      "name": "Yarkoni, Tal"
+    },
+    {
+      "affiliation": "University of Texas at Austin",
+      "name": "de la Vega, Alejandro",
+      "orcid": "0000-0001-9062-3778"
+    },
+    {
+      "affiliation": "Stanford University",
+      "name": "Markiewicz, Christopher J.",
+      "orcid": "0000-0002-6533-164X"
+    },
+    {
+      "affiliation": "Department of Psychology, Stanford University",
+      "name": "Gorgolewski, Krzysztof J.",
+      "orcid": "0000-0003-3321-7583"
+    },
+    {
+      "affiliation": "Florida International University",
+      "name": "Salo, Taylor",
+      "orcid": "0000-0001-9813-3167"
+    },
+    {
+      "name": "DeStasio, Krista"
+    },
+    {
+      "affiliation": "Dartmouth College",
+      "name": "Halchenko, Yaroslav O.",
+      "orcid": "0000-0003-3456-2493"
+    },
+    {
+      "name": "Petrov, Dmitry"
+    },
+    {
+      "affiliation": "National Institute of Mental Health",
+      "name": "Nielson, Dylan M.",
+      "orcid": "0000-0003-4613-6643"
+    },
+    {
+      "affiliation": "Montreal Neurological Institute and Hospital",
+      "name": "Kiar, Gregory",
+      "orcid": "0000-0001-8915-496X"
+    },
+    {
+      "name": "Whitaker, Kirstie",
+      "orcid": "0000-0001-8498-4059"
+    },
+    {
+      "affiliation": "Montreal Neurological Institute and Hospital",
+      "name": "DuPre, Elizabeth",
+      "orcid": "0000-0003-1358-196X"
+    },
+    {
+      "name": "Ivanov, Alexander"
+    },
+    {
+      "affiliation": "CorticoMetrics LLC",
+      "name": "Tirrell, Lee S.",
+      "orcid": "0000-0001-9393-8361"
+    },
+    {
+      "affiliation": "Otto-von-Guericke-University Magdeburg, Germany",
+      "name": "Hanke, Michael",
+      "orcid": "0000-0001-6398-6370"
+    },
+    {
+      "affiliation": "T\u00e9l\u00e9com ParisTech, France",
+      "name": "Jas, Mainak",
+      "orcid": "0000-0002-3199-9027"
+    },
+    {
+      "affiliation": "Department of Psychology, Stanford University",
+      "name": "Esteban, Oscar",
+      "orcid": "0000-0001-8435-6191"
+    },
+    {
+      "name": "Staden, Isla",
+      "orcid": "0000-0002-0795-1154"
+    },
+    {
+      "name": "Carlin, Johan"
+    },
+    {
+      "affiliation": "The University of Washington eScience Institute",
+      "name": "Rokem, Ariel",
+      "orcid": "0000-0003-0679-1985"
```

Comment:
@arokem

Reply:
That's correct. 

I do believe that my contribution to this work (once you've removed the shablona part of the history) is [a single character change in the README](https://github.com/bids-standard/pybids/pull/238/files), but I'll take it! 😄